src/services.jl

```diff
@@ -90,3 +90,11 @@ function wait_for_service(service::AbstractString; kwargs...)
         error("Timeout exceeded waiting for service '$service'")
     end
 end
+
+"""
+    shutdown(service_obj)
+Shut down the specified service.
+"""
+function shutdown(s::Service{ST}) where ST <: AbstractService
+    pycall(s.srv_obj["shutdown"], Nothing)
```

Comment:
PyCall has deprecated this style of accessing functions, the preferred style is simply `s.srv_obj.shutdown`

Reply:
Thanks for your review! I fixed the style.